cli/cli_util.c

```diff
@@ -115,10 +120,30 @@ kvdb_compact_request(const char *kvdb_home, const char *request_type, u32 timeou
     uint   sleep_secs = 2;
     char **namev;
     size_t namec;
+    size_t kvdb_paramc = 0;
+    const char *kvdb_paramv[1] = { 0 };
+    size_t kvs_paramc = 0;
+    const char *kvs_paramv[1] = { 0 };
+
+
+    /* Full compactions need a few extra params.
+     */
+    if (request == req_compact_full) {
+        kvdb_paramv[kvdb_paramc++] = "csched_full_compact=true";
+        assert(kvdb_paramc <= NELEM(kvdb_paramv));
+        kvs_paramv[kvs_paramc++] = "cn_close_wait=true";
+        assert(kvs_paramc <= NELEM(kvs_paramv));
+    }
```

Comment:
@beckerg see here.  `cn_close_wait` is only set for full compact requests.  it was the same on the patch you were looking at too, but it was less obvious than it is w/ this update.